farcy/__init__.py

```diff
@@ -326,6 +335,9 @@ def handle_pr(self, pr):
                     info = pr.create_review_comment(*args).html_url['href']
                 self.log.info('PR#{0} ({1}:{2}) COMMENT: "{3}"'.format(
                     pr.number, pfile.filename, lineno, info))
+                comments_added += 1
```

Comment:
Why is it that the previous conditional does `continue` and this one does `break`? Should only the former one do `break`, or should we always find out how many issues there are?